frameworks/Ruby/rails/rails-contrast-assess.dockerfile

```diff
@@ -0,0 +1,29 @@
+# Ruby Contrast Agent doesn't support 3.0 yet
+FROM ruby:2.7
+
+ENV BUNDLE_WITHOUT=mysql
+ENV RAILS_ENV=production_postgresql
+ENV PORT=8080
+
+EXPOSE 8080
+WORKDIR /rails
+
+COPY ./Gemfile* /rails/
+
+RUN bundle install --jobs=8
+
+COPY . /rails/
+
+# Start Contrast Additions
+COPY contrast-agent.gem contrast-agent.gem
+COPY contrast_security.yaml /etc/contrast/contrast_security.yaml
+
+ENV CONTRAST__ASSESS__ENABLE=true
```

Comment:
I'm assuming this is because of the TechEmpower runner, but I'll ask anyway since I've noticed this pattern: Is there a reason we can't pass these environment variables in at execution time? I know Docker allows it, but again, I assume we don't have that flexibility with their runner.

Reply:
The way the runner is designed, each framework variant needs to be enumerated in benchmark_config.json, and have a corresponding docker file. There's no real way to pass in arguments to the docker run command that's used.